doc/ug/github_notes.md

```diff
@@ -79,9 +79,9 @@ $ git config user.email "my_name@lowrisc.org"
 ## Working in your local repo
 
 The repository that you have created locally will initially be on the
-**master** branch. In general you should not make changes on this
+`master` branch. In general you should not make changes on this
 branch, just use it to track your GitHub repository and synchronize with the
-lowRISC master repo.
+lowRISC `master` repo.
```

Comment:
I think I understand how this patch is working: the "master" branch has to keep its name, but we're monospacing it to make it clear this is a technical term. This line refers to the repository, though. I think `synchronize with the main lowRISC repository` would be equivalent.